docs/flashbots-auction/advanced/rpc-endpoint.mdx

```diff
@@ -768,6 +768,76 @@ where
 - `consideredByBuildersAt`: indicates time at which each builder selected the bundle to be included in the target block
 - `sealedByBuildersAt`: indicates time at which each builder sealed a block containing the bundle
 
+### flashbots_getSbundleStats
+
+The `flashbots_getSbundleStats` JSON-RPC method returns stats for a single sbundle. You must provide bundleHash, and the signing address must be the same as the one who submitted the bundle.
+Only bundles not containing backruns are supported
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "flashbots_getSbundleStats",
+  "params": [
+    {
+      bundleHash,       // String, returned by the flashbots api when calling eth_sendBundle with builders field or mev_sendBundle
```

Comment:
Is this returned in addition to or instead of the `smart` field? Currently our docs say that a `smart` field is returned when calling `eth_sendBundle` with builders[]

Reply:
Yes it is returned together with `eth_sendBundle` response or else it will be hard for searcher to use stats api. So `eth_sendBundle` api is backward compatible, `bundleHash` is always returned, in case `smart:true` returned, searcher should query this endpoint.